frontend/src/components/NavBar/navBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './navBar.module.css'
+
+function NavBar() {
+    return(
+        <div className={styles["fake-body"]}>
```

Comment:
No idea what this means, was this 'div' tag supposed to override the body tag's style because the body tag's style was affecting your work?

Reply:
Yeah I think I forgot to remove it after but it should be fixed now